test/test-cases/scale/saic/test_sai_vnet_outbound_simple.py

```diff
@@ -242,9 +241,19 @@ def test_vnet_inbound_simple_packet_modification(self, dpu, dataplane):
 
         print("\nSending outbound packet...\n\n", vxlan_pkt.__repr__())
         send_packet(dataplane, 0, vxlan_pkt)
-        time.sleep(0.5)
+        time.sleep(5)
         print("\nVerifying packet...\n", vxlan_exp_pkt.__repr__())
-        verify_packet(dataplane, vxlan_exp_pkt, 0)
+        print("packet stats......................................")
+        rows = dataplane.get_all_stats()
+        print("{}".format(rows[0].name))
```

Comment:
probably we can leave this file as is and not modify it, and create 1 or 2... other tests that have focus on adding few routes and than sending flows over those to see if route gets respected, if no route default action is to reflect traffic back

Reply:
Yes reverted back,